--- a/src/Main/Main.make	2015-08-12 15:37:07.747377368 +0100
+++ b/src/Main/Main.make.new	2015-08-12 15:37:17.089368142 +0100
@@ -132,7 +132,7 @@
 
 $(APPNAME): $(LIBS) $(OBJS)
 	@echo Linking $@
-	$(CXX) -o $(APPNAME) $(OBJS) $(LIBS) $(FUSE_LIBS) $(WX_LIBS) $(LFLAGS)
+	$(CXX) -o $(APPNAME) $(OBJS) $(LIBS) $(FUSE_LIBS) $(WX_LIBS) $(LFLAGS) -ldl
 
 ifeq "$(TC_BUILD_CONFIG)" "Release"
 ifndef NOSTRIP
